sandbox/public/index.html

```diff
@@ -41,6 +42,7 @@
         propertyId: 9999999,
         imsOrgId: "53A16ACB5CC1D3760A495C99@AdobeOrg",
         log: true,
+        authoringMode: document.location.href.indexOf("mboxEdit") !== -1,
```

Comment:
Can you pls add a comment here to explain the reason why we need the `mboxEdit` voodoo?

Reply:
Done